README.md

```diff
@@ -1 +1,34 @@
-# Portfolio-Setup-and-Mobile-firs
\ No newline at end of file
+# Portfolio-Setup-and-Mobile-firs
+
+Description of the project:
+
+-Set up a new repository and prepare it for development using best practices (e.g. linters).
+-Build the first 2 sections of the mobile website using the template'
+
+Built with 
+        Javascript
+        HTML
+        CSS
+
+General Requirements:
+
+    Make sure that there are no linter errors
+    Make sure that you used correct GitHub Flow
```

Comment:
- [ ] It is strongly recommended to use Microverse provided [README template ](https://github.com/microverseinc/readme-template/blob/master/README.md)while writing the README files for our projects. This helps you write a professional README that conveys all the necessary info about your project in a neat way. Currently, it is not looking professional so please try to use that template provided and then customize it to be specific to your project. 😉

Reply:
Rectified